test/sagemaker_tests/tensorflow/tensorflow2_training/requirements.txt

```diff
@@ -6,7 +6,7 @@ Pillow
 h5py
 tox
 flake8
-pytest==5.3.5
+pytest
 pytest-cov
 pytest-rerunfailures==9.0
 pytest-xdist
```

Comment:
If pytest-rerunfailures is pinned to a version, then pytest should also be pinned to a version.

Please upgrade that pinned version if necessary, but don't unpin it as it causes issues with test setup when other pytest packages are updated.

Reply:
I will pin it with recent version in this case.